remote/remote_test.go

```diff
@@ -1569,3 +1530,11 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 		})
 	})
 }
+
+func VerifyRetryLogic(t *testing.T, mockServer *h.MockServer, repoName string, expectedCount int) {
```

Comment:
It doesn't have to be public. You can change it to `verifyRetryLogic`.

Reply:
Oh. Sorry, I didn't notice the difference between the capital letter or not during the method definition.  I just googled it, I agreed it should be not exported.